docs/rules/camelcase.md

```diff
@@ -60,6 +78,19 @@ var { category_id: category } = query;
 
 ### never
 
+
+Examples of **incorrect** code for this rule with the default `{ "properties": "never" }` option:
```

Comment:
I don't think we need this or this change any more, since this behavior is the same as the cases listed above.